azext_iot/central/params.py

```diff
@@ -654,3 +661,261 @@ def load_central_arguments(self, _):
+        )
+        context.argument(
+            "enabled",
+            options_list=["--enabled"],
+            arg_type=get_three_state_flag(),
+            help="Whether the devices using the group are allowed to connect to IoT Central. True or False."
+        )
+        context.argument(
+            "certificate",
+            options_list=["--certificate", "--cert"],
+            help="The string representation of this x509 certificate.",
+        )
+        context.argument(
+            "verified",
+            options_list=["--verified", "--v"],
+            arg_type=get_three_state_flag(),
+            help="Whether the x509 certificate has been verified. True or False.",
+        )
+        context.argument(
+            "remove_x509",
+            options_list=["--remove-x509"],
+            arg_type=get_three_state_flag(),
+            help="Whether the x509 certificate should be removed from the group. True or False."
+        )
+        context.argument(
+            "etag",
+            options_list=["--etag"],
+            help="ETag used to prevent conflict in enrollment group updates."
+        )
+
+    with self.argument_context("iot central enrollment-group verify-x509") as context:
+        context.argument(
+            "certificate",
+            options_list=["--certificate", "--cert"],
+            help="The string representation of this x509 certificate.",
+        )
+
+    with self.argument_context("iot central scheduled-job") as context:
+        context.argument(
+            "job_id",
+            options_list=["--job-id", "--id"],
+            help="Unique identifier for the scheduled job.",
+        )
+
+    with self.argument_context("iot central scheduled-job create") as context:
+        context.argument(
+            "job_name",
+            options_list=["--job-name"],
+            help="Display name of the job.",
+        )
+        context.argument(
+            "group_id",
+            options_list=["--group-id", "-g"],
+            help="The ID of the device group on which to execute the job",
+        )
+        context.argument(
+            "schedule",
+            options_list=["--schedule"],
+            help="The schedule at which to execute the job. Provide path to JSON file or raw stringified JSON."
+            " [File Path Example:./path/to/file.json]"
+            " [Example of stringified JSON:[{<Schedule Data JSON>}].",
+        )
+        context.argument(
+            "content",
+            options_list=["--content", "-k"],
+            help="The job data definition. Provide path to JSON file or raw stringified JSON."
+            " [File Path Example:./path/to/file.json]"
+            " [Example of stringified JSON:[{<Job Data JSON>}]. The request body must contain array of JobData.",
+        )
+        context.argument(
+            "batch_type",
+            options_list=["--batch-type", "--bt"],
+            choices=CaseInsensitiveList(["number", "percentage"]),
```

Comment:
you can add this as an enum and have the param use the enum (see `az iot dps enrollment-group create --reprovision-policy`)

Reply:
Okay